sdk/python/kfp_tekton/compiler/compiler.py

```diff
@@ -63,11 +118,12 @@ def _create_dag_templates(self, pipeline, op_transformers=None, params=None, op_
     for op in pipeline.ops.values():
       for transformer in op_transformers or []:
         transformer(op)
+
     tasks = []
     for op in pipeline.ops.values():
       tasks.extend(op_to_steps_handler(op))
 
-    return tasks
+    return tasks, root_group
```

Comment:
we should not change function signatures/return types unless we have to

please revert
```Python
     return tasks, root_group 
```
back to
```Python
     return tasks
```

...and in `_create_pipeline_workflow` on line 143 see comment below